includes/admin.form.inc

```diff
@@ -54,53 +73,64 @@ function islandora_fits_admin_form($form, $form_state) {
 }
 
 /**
+ * Validation handler for Fits admin form.
+ */
+function islandora_fits_admin_form_validate($form, &$form_state) {
+  if ($form_state['values']['islandora_fits_do_derivatives']) {
+    if (!islandora_fits_path_check($form_state['values']['islandora_fits_executable_path'])) {
+      form_set_error('islandora_fits_path_textfield', "Path to FITS is not valid. Provide a valid path or disable FITS derivatives.");
+    }
+  }
+  else {
+    // TODO: Clear the fits executable path?
```

Comment:
I would not adjust the FITS path, incase someone is just toggling this setting short term.

Reply:
Cool. My worry was that it could remain set (though invalid) and eventually get called, since we're only testing for validity when saving the form and not when executing the derivatives.

Reply:
So ideally it should not get set anymore.

But for handling old broken paths you could add a `hook_update_N()` that runs the test and if it fails, unsets the path and sets `islandora_fits_do_derivatives` to `FALSE`. 

But you'll need to clearly explain that is happening in the `hook_update_N` function description (what is shown to the user)

Reply:
I kinda like that idea. What do @Islandora/7-x-1-x-committers think of a hook_update_N to unset invalid paths?

Reply:
I have NOT implemented a hook to remove invalid paths. If someone has a current path, that is invalid, then they will not be able to save the FITS form without either changing it to a valid path, or disabling derivatives.